src/qt/bitcoingui.cpp

```diff
@@ -681,6 +704,54 @@ void BitcoinGUI::openClicked()
     }
 }
 
+void BitcoinGUI::newWallet()
+{
+    QString file_name = QFileDialog::getSaveFileName(this, tr("New Wallet"));
+    if (file_name.isEmpty()) return;
+
+    std::string error, warning;
+    if (!m_node.createWallet(QDir::toNativeSeparators(file_name).toStdString(), error, warning)) {
+        QMessageBox::information(this, tr("New Wallet"), QString::fromStdString(error));
+        return;
+    }
+    if (!warning.empty()) {
+        QMessageBox::information(this, tr("New Wallet"), QString::fromStdString(warning));
+    }
+}
+
+void BitcoinGUI::openWallet()
+{
+    QString file_name = QFileDialog::getExistingDirectory(this, tr("Open Wallet"), QString(), QFileDialog::DontResolveSymlinks);
```

Comment:
Why is this `getExistingDirectory()`? A wallet can also be a file I think.

Also defaults to the directory that qt is running in.